sonar-db/src/main/java/org/sonar/db/permission/PermissionQuery.java

```diff
@@ -127,79 +119,72 @@ public static Builder builder() {
 
   public static class Builder {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)